### PR TITLE
Add OAK for Education page with Cortic AI Toolkit mention

### PR DESCRIPTION
**Pull request by VanDavv**
Screenshot 2021-08-24 at 15 24 07

### CODE REVIEW

**Comment by Luxonis-Brandon** [comment]
Behind so I don't think will have a proper review but could we add this GIF on the page?

https://github.com/cortictechnology/cep/blob/main/resources/Visual-Programming.gif

It's very compelling to show what it does.

**Comment by VanDavv** [comment]
Thanks @Luxonis-Brandon, agreed. Just updated the PR
image

File: source/pages/education.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+by visiting this project on `GitHub <https://github.com/cortictechnology/cait>`__
+
+Kudos to the Cortic AI team for sharing their awesome tool!
+
+
+Looking for more?
+#################
+
+You can also check out our :code:`#community` channel on our `Discord server <https://discord.gg/EPsZHkg9Nx>`__, where you can
+meet people behind these projects or share your own idea / demo - we're looking forward to meet you there!
```
Comment (Erol444) [inline source/pages/education.rst:37]: to meet**ing** you there

File: source/pages/education.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    <div style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+        <iframe src="https://www.youtube.com/embed/YB8UcSVhopM" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
+    </div>
+    <br/>
+    <a href="https://github.com/cortictechnology/cait" target="_blank" class="btn item-cta">Available on GitHub!</a>
+    <p/>
+
+The Cortic A.I. Toolkit is a software package that enables makers and students to learn and experiment with A.I.
+algorithms on the popular Raspberry Pi 4B single board computer. Provides a gentle introduction to A.I. for
```
Comment (Erol444) [inline source/pages/education.rst:20]: **It** provides...

File: source/pages/education.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+beginners while offering a lot of customization possibilities and depth so that users can progressively grow with it
```
Comment (Erol444) [inline source/pages/education.rst:21]: and depth **perception/estimation**.. maybe?

**Review by Erol444 — APPROVED** [review]
Few minor typos/nitpicks, LGTM!

**Pull request merged**